docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -769,3 +771,31 @@ CREATE TABLE ASN_SUBMISSION_SUBMITTER (
     CONSTRAINT UK_FHL15YNESBCTBUS4859J78D8F UNIQUE(SUBMISSION_ID,SUBMITTER)
 );
 
+ALTER TABLE ASN_PEER_ASSESSMENT_ITEM_T MODIFY REVIEW_COMMENT longtext;
+
+-- END SAK-32101
+
+-- SAK-33896 Remove this site association table
+drop table IF EXISTS SITEASSOC_CONTEXT_ASSOCIATION;
+-- END SAK-33896
+
+-- KNL-945 Hibernate changes
+ALTER TABLE CHAT2_MESSAGE
+    DROP FOREIGN KEY FK720F9882555E0B79;
+    MODIFY CHANNEL_ID varchar(36)
+    MODIFY MESSAGE_ID varchar(36);
```

Comment:
this looks wrong ....

Reply:
Well, I'm assuming that all FK's are named the same and that it's generated based on some naming algorithm If this isn't the case it's either going to have to.

- Figure out what named FK to drop (I had an idea for this but it wasn't working right so I need a better idea)
- Change the size in the code back up to 99
- Just not bother changing the size from 99 to 36

Suggestions accepted!

Reply:
you have semicolon ending line 784 .... meant to be a comma?

Reply:
Well that part is certainly wrong! :) What do you think about the FK name? I was hoping it's okay, maybe I should check a few Monday.